openpower/package/openpower-pnor/0001-Increase-BOOTKERNEL-to-24-MB.patch

```diff
@@ -0,0 +1,185 @@
+From f4bcde7cc8a7b5c298c71b162faf11c5eb2e613b Mon Sep 17 00:00:00 2001
```

Comment:
This patch modifies things for all configurations, which may not be the intent.

At some point the endless expanding of BOOTKERNEL should cease and instead look at how the mtd device could be used through dm-verity or similar, likely reducing IO on boot at the same time as allowing for expansion.

Reply:
Yes, this patch is an RFC only and I don't intend to merge it as is (so that the user in question could use it as a reference in how to do it local to their build in a sustainable way).